src/stdlib/map_util.bpf.c

```diff
@@ -0,0 +1,11 @@
+#define __KERNEL__
```

Comment:
One other nit: let's try to avoid naming anything `util` as long as possible. I recognize that it was a pre-existing thing in the main codebase, but in general I think it attracts a kitchen sink of stuff. Perhaps we can just call it `maps.bpf.c` instead.

(See the statement from the authors of Go on this, which I agree with quite a lot: https://go.dev/blog/package-names#bad-package-names)

Reply:
BTW, good bit from the Go blog.